libraries/AP_LandingGear/AP_LandingGear.cpp

```diff
@@ -63,6 +64,24 @@ const AP_Param::GroupInfo AP_LandingGear::var_info[] = {
     // @Values: 0:Low,1:High
     // @User: Standard
     AP_GROUPINFO("WOW_POL", 6, AP_LandingGear, _pin_weight_on_wheels_polarity, DEFAULT_PIN_WOW_POL),
+
+    // @Param: DEPLOY_ALT
+    // @DisplayName: Deployment altitude
```

Comment:
I think we should have the subsystem at the beginning of the Display Name.  so instead of "Deployment altitude" it should be "Landing Gear deployment altitude".  The reason is that otherwise the user can't determine the subsystem without looking at the parameter name itself.  In some ground stations the Display name appears before the parameter name so it will be more quickly understood by the user if the subsystem name appears first

Reply:
done